app/controllers/concerns/unconfirmed_user_concern.rb

```diff
@@ -25,11 +25,21 @@ def track_user_already_confirmed_event
       errors: { email: [t('errors.messages.already_confirmed')] },
       user_id: @user.uuid,
     )
+    irs_attempts_api_tracker.user_registration_email_confirmation(
+      email: @email_address.email,
+      success: false,
+      failure_reason: "email #{t('errors.messages.already_confirmed')}",
```

Comment:
Hi just to reiterate [my point from earlier](https://github.com/18F/identity-idp/pull/6674#discussion_r935747567), I strongly think we should avoid translated strings inside these failure reasons because they'll be inconsistent across locales.

A few options:

1. (my preferred option): bring back the hash structure and use info from `error_details` only:
```suggestion
      failure_reason: { email: "already_confirmed" }
```
2. use string slugs like 
```suggestion
      failure_reason: "email_already_confirmed",
```

Reply:
I like option 1 which you preferred - so then we will just need to use the already error_details instead of creating our new method, but also we need to change this failure reason key from string to hash in our schema doc. Let me know your thoughts.

Reply:
@zachmargolis Do you think we should update the schema to reflect that we'll return a hash/array, then? I'd kind of waffled on this yesterday.

Reply:
Yeah I think it's worth a schema update to use a Hash? Because if I was a consumer of the API, either of those are easier than trying to match on data inside a string?